ynput/OpenPype
# Max: Add missing workfile creator

## Changelog Description
Add the missing workfile creator in 3dsMax.

## Additional info
For some reason, `get_subset_name` function can't set the name with the correct variant, although there is variant.
It is using the old subset name `workfileAsset`. 
## Testing notes:
1. Launch Max via launcher
2. Load Scene
3. Open publisher
4. it should automatically create the workfile instance
5. you can notice the new container (which is hidden and greyed out) named accordingly
6. Publish workfile
7. It should be successfully published.
